totalRP3/modules/register/characters/register_characteristics.lua

```diff
@@ -448,13 +446,25 @@ local function saveInDraft()
 		local psychoLine = psychoEditCharFrame[index];
 		psychoStructure.V2 = psychoLine.V2;
 
+		-- Clear out the colors prior to persistence.
+		psychoStructure.LC = nil;
+		psychoStructure.RC = nil;
+
 		if not psychoStructure.ID then
 			-- If not a preset
 			psychoStructure.LT = stEtN(psychoLine.CustomLeftField:GetText()) or loc("REG_PLAYER_LEFTTRAIT");
 			psychoStructure.RT = stEtN(psychoLine.CustomRightField:GetText()) or loc("REG_PLAYER_RIGHTTRAIT");
 
-			psychoStructure.LC = stEtN(psychoLine.LC);
-			psychoStructure.RC = stEtN(psychoLine.RC);
+
+			local lc = psychoLine.LC;
+			if lc then
+				psychoStructure.LC = { r = lc:GetRed(), g = lc:GetGreen(), b = lc:GetBlue() };

```

Comment:
I'm gonna need to make a color:GetRGBTable() method to help with this kind of usage 🤔

Reply:
Yeah, was thinking that it's not the prettiest. I can store it as hex codes again - I'm probably the only one storing with the new format right now after all!

Reply:
I have added the `color:GetRGBTable()` and `color:GetRGBATable()` methods to Ellyb and I have update the submodules in Total RP 3 to include the latest changes. You can merge the development branch into your branch to get this and use it :)